HPO/main.py

```diff
@@ -3,23 +3,80 @@
 import dask
 from dask import delayed
 from dask_cuda import LocalCUDACluster
+from dask_kubernetes import KubeCluster
 from dask.distributed import Client
 
+from time import sleep # sleep needed to give K8S time for manual scaling
 import argparse
 
-def launch_dask(n_gpus):
-    cluster = LocalCUDACluster(ip="", n_workers=n_gpus)
+
+default_worker_spec_fname = '/worker_spec.yaml'
+default_worker_spec = '''
+# worker-spec.yml
+
+kind: Pod
+metadata:
+  labels:
+    foo: bar
+spec:
+  restartPolicy: Never
+  containers:
+  - image: supertetelman/k8s-rapids-dask:0.9-cuda10.0-runtime-ubuntu18.04
+    imagePullPolicy: IfNotPresent
+    args: [dask-worker,  --nthreads, '1', --no-bokeh, --memory-limit, 6GB, --no-bokeh, --death-timeout, '60']
+    name: dask
+    resources:
+      limits:
+        cpu: "2"
+        memory: 6G
+        nvidia.com/gpu: 1
+      requests:
+        cpu: "2"
+        memory: 6G
+        nvidia.com/gpu: 1
+    env:
+      - name: PRE_RUN_HOOK
+        value: "pip install -e git+https://github.com/supertetelman/rapids.git@k8s#egg=rapids&subdirectory=HPO"
```

Comment:
This section needs to be updated.

Reply:
We decided just to move this bit into the Dockerfile (in this repo and the supertetelman/k8s-rapids-dask repo)